packages/core/src/components/DataTable/helpers/sortData.ts

```diff
@@ -1,20 +1,26 @@
 import { SortDirection, SortDirectionType } from 'react-virtualized';
 import { GenericRow } from '../types';
 
-function sortDesc(a: any, b: any) {
-  if (typeof b === 'undefined' || a < b) {
+// https://stackoverflow.com/questions/29829205/sort-an-array-so-that-null-values-always-come-last
+function sort(a: any, b: any, ascending: boolean = false) {
+  // Equal items sort equally.
+  if (a === b) {
+    return 0;
+  }
+  // null sort after anything else.
+  if (a == null || a == 'undefined') {
```

Comment:
if `a` is `null` or `undefined` both of these double equals `==` checks return true - redundant. personally, i try to always use triple equals checks. is this truly sorting null to come last over undefined?

<img width="210" alt="Screen Shot 2019-08-12 at 4 23 52 PM" src="https://user-images.githubusercontent.com/306275/62905055-aca97a00-bd1d-11e9-935a-fa7c27f3c277.png">

the stackoverflow answer you linked also used `=== null` check

Reply:
I haven't thought about null/undefined sorting relative to each other. Just updated it to use `  if (b == null || b === 'undefined') {`

`=== null` doesn't work in the storybook, 
http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html

Reply:
so, to be more explicit with the checks:

`a === null || typeof a === 'undefined'`
`b === null || typeof b === 'undefined'`

Reply:
this'll sort `var c = ['foo', null, 'bar', null, undefined, 'baz', undefined]` to `["bar", "baz", "foo", null, undefined]`
(example minus the asc/desc logic):

<img width="439" alt="Screen Shot 2019-08-12 at 4 44 46 PM" src="https://user-images.githubusercontent.com/306275/62905717-89cc9500-bd20-11e9-91f6-19da00580848.png">

is that what you're expecting?

Reply:
to check if it is `undefined` you have to do `typeof b ==='undefined'`, doing just `b === 'undefined'` won't return the proper boolean result.

dang it storybook! knowing that it won't all `=== null`, you can reduce the two checks to be just `b == null` (no `typeof b === 'undefined'`)

so just `if (a == null) return 1;` and `if (b == null) return -1`

Reply:
Awesome this works, thank you so much for helping me think through this!